dfget/core/downloader/power_client.go

```diff
@@ -158,6 +161,7 @@ type ClientWriter struct {
 	targetQueue  util.Queue
 	targetWriter *TargetWriter
 
+	api api.SupernodeAPI
```

Comment:
Is it necessary? In fact, I think it's more reasonable to set `api.SupernodeAPI` as a global variable, what do you think?

Reply:
It will be easier for testing to set `api.SupernodeAPI` as a field of `ClientWriter` than as a global variable.